addons/pos_restaurant/static/src/js/multiprint.js

```diff
@@ -169,6 +169,24 @@ screens.OrderWidget.include({
 
 var _super_order = models.Order.prototype;
 models.Order = models.Order.extend({
+    build_by_product_resume: function(){
+        var resume = {};
+
+        this.orderlines.each(function(line){
+          var pid = line.get_product().id;
```

Comment:
if mp_skip is true, you are not suppose to add content to the resume, because you don't want to send it. it is the purpose of the fix.